docker/docker-compose.ci.yml

```diff
@@ -73,6 +73,22 @@ services:
       PAYWALL_SCRIPT_URL: '${PAYWALL_SCRIPT_URL}'
       UNLOCK_STATIC_URL: '${UNLOCK_STATIC_URL}'
     command: ['npm', 'run', 'start']
+  paywall-integration:
```

Comment:
this is identical to paywall, but will require updating the network to `paywall-integration` everywhere `paywall` was referenced (which the rest of this PR does)